README.md

```diff
@@ -4,4 +4,93 @@
 
 This repo will contain standard global configurations for ACCESS-ESM1.5, the ACCESS Coupled Earth System Model.
 
+This is an "omnibus repository": it contains multiple related configurations, and each
+configuration is stored in a separate branch.
+
+Branches utilise a simple naming scheme:
+
+`{release}-{scenario}[+{modifier}]`
+
+where `release` signifies this is the release branch that is tested, versioned and ready for use, `scenario` is the base experimental design with optional `modifiers`. All configurations are assumed to be global extent with nominal 1 degree resolution.
+
+Some examples of possible values of the specifiers:
+
+* scenario: historical, preindustrial, ssp126
+* modifier: concentration, interactiveC, noLUC
+
+where scenario is typically a [CMIP experiment identifier](https://wcrp-cmip.github.io/CMIP6_CVs/docs/CMIP6_experiment_id.html), concentration and interactiveC describe the CO2 cycling protocol, and noLUC is no land-use change.
+
 Most configurations are adapted from work by the [CLEX CMS team](https://github.com/coecms/access-esm).
+
+## Supported configurations
+
+All available configurations are browsable under [the list of branches](https://github.com/ACCESS-NRI/access-esm1.5-configs/branches) and should also be listed below:
+
+| Branch | Configuration Description |
+| ------ | ------------------------- |
+
+There are more detailed notes contained in the respective branches for each configuration.
+
+More supported configurations will be added over time.
+
+## How to use this repository to run a model
+
+All configurations use [payu](https://github.com/payu-org/payu) to run the model.
+
+This repository contains many related experimental configurations to make support
+and discovery easier. As a user it does not necessarily make sense to clone all the
+configurations at once.
+
+In most cases only a single experiment is required. If that is the case choose which experiment and then run
+
+```sh
+git clone -b <experiment> https://github.com/ACCESS-NRI/model-configs/ <experiment>
+```
+
+and replace `<experiment>` with the branch name or tag of the experiment you wish to run.
+
+[ACCESS-Hive](https://access-hive.org.au/) contains [detailed instructions for how to configure and run `ACCESS-ESM1.5` with `payu`](https://access-hive.org.au/models/run-a-model).
+
+## CI and Reproducibility Checks
+
+This repository makes use of GitHub Actions to perform reproducibility checks on model config branches.
+
+### Config Branches
+
+Config branches are branches that store model configurations of the form: `release-<config>` or `dev-<config>`, for example: `release-1deg_jra55_iaf`. For more information on creating your own config branches, or for understanding the PR process in this repository, see the [CONTRIBUTING.md](CONTRIBUTING.md).
```

Comment:
```suggestion
Config branches are branches that store model configurations of the form: `release-<config>` or `dev-<config>`, for example: `release-historical+concentration`. For more information on creating your own config branches, or for understanding the PR process in this repository, see the [CONTRIBUTING.md](CONTRIBUTING.md).
```

Reply:
Done in https://github.com/ACCESS-NRI/access-esm1.5-configs/pull/18/commits/dfd9512fd715542994ebf48f6b996c0b98f83d45